src/CoreWCF.WebHttp/src/Resources/Strings.resx

```diff
@@ -375,6 +375,9 @@
   <data name="SerializingRequestNotSupportedByFormatter" xml:space="preserve">
     <value>The formatter '{0}' does not support serializing requests.</value>
   </data>
+  <data name="ServiceModelWebServiceNotRegistered" xml:space="preserve">
```

Comment:
Small typo on the resource name, WebServices should be pluralized. E.g. `ServiceModelWebServicesNotRegistered`